overlord/boot/firstboot_test.go

```diff
@@ -134,3 +160,145 @@ snaps:
 	c.Assert(err, IsNil)
 	c.Assert(snapst.DevMode(), Equals, true)
 }
+
+func (s *FirstBootTestSuite) makeModelAssertion(c *C, modelStr string) *asserts.Model {
+	headers := map[string]interface{}{
+		"series":       "16",
+		"authority-id": "my-brand",
+		"brand-id":     "my-brand",
+		"model":        modelStr,
+		"class":        "my-class",
+		"architecture": "amd64",
+		"store":        "canonical",
+		"gadget":       "pc",
+		"kernel":       "pc-kernel",
+		"core":         "core",
+		"timestamp":    time.Now().Format(time.RFC3339),
+	}
+	model, err := s.brandSigning.Sign(asserts.ModelType, headers, nil, "")
+	c.Assert(err, IsNil)
+	return model.(*asserts.Model)
+}
+
+func (s *FirstBootTestSuite) makeModelAssertionChain(c *C) []asserts.Assertion {
+	assertChain := []asserts.Assertion{}
+
+	brandAcct := assertstest.NewAccount(s.storeSigning, "my-brand", map[string]interface{}{
+		"account-id":   "my-brand",
+		"verification": "certified",
+	}, "")
+	s.storeSigning.Add(brandAcct)
```

Comment:
fyi, this doesn't hurt and is a good double check that the chain is sane but you don't need to add this to the storeSigning db unless we need to retrieve things against it, same below